src/test/e2e/00_use_cli_test.go

```diff
@@ -82,5 +84,26 @@ func TestUseCLI(t *testing.T) {
 	require.NoError(t, err, "Error when reading image cache path")
 	assert.Greater(t, len(files), 1)
 
-	e2e.cleanFiles(shasumTestFilePath, imageCachePath, otherTmpPath)
+	pkgName = fmt.Sprintf("zarf-package-git-data-%s.tar.zst", e2e.arch)
+
+	// Pull once to test git cloning
+	stdOut, stdErr, err = e2e.execZarfCommand("package", "create", "examples/git-data", "--confirm", "--zarf-cache", cachePath, "--tmpdir", otherTmpPath, "--log-level=debug")
+	require.NoError(t, err, stdOut, stdErr)
+
+	files, err = os.ReadDir(gitCachePath)
+	require.NoError(t, err, "Error when reading git cache path")
+	assert.Greater(t, len(files), 1)
+
+	// Pull twice to test git fetching (from cache)
+	stdOut, stdErr, err = e2e.execZarfCommand("package", "create", "examples/git-data", "--confirm", "--zarf-cache", cachePath, "--tmpdir", otherTmpPath, "--log-level=debug")
```

Comment:
Do we have any way to verify that Zarf is actually using the repo cache instead of pulling again?

Reply:
We test that the gitCache directory gets populated, and you can't git clone into an already existing folder, though I suppose in code we could just delete the folder and reclone...  Not sure how to test that...